### PR TITLE
Add React view for Tutorials

### PR DESCRIPTION
**Pull request by james-wallis**
**Draft until I fix the React tests**
### Summary
Converted the TutorialView from opening the tutorials in markdown preview mode to having their own React view in order to:
* Remove bugs described in #2645 
* Removed duplicated data stored in Tutorials file + tutorial.json (title, read time, etc)
* Moved the `tutorials` directory to under `resources` so that VSCode can access the images, also moved `tutorials.json` to the same folder to keep the tutorial stuff together. (I've updated the PDF location for downloads)
* Converted most of the `markdown -> pdf` to JS as I needed to change it to cope with the changes to the tutorials (title no longer kept in tutorial md file) 

Added the following NPM packages for markdown to html:
```
"remark-html": "^13.0.1",
"remark-parse": "^8.0.3",
"unified": "^9.2.0",
"react-html-parser"
```

### Testing 
* Modified the test for `TutorialView.ts`.
* Visually tested the css.

### Screenshots
image
image

### CODE REVIEW

File: packages/blockchain-ui/src/components/pages/TutorialSinglePage/TutorialSinglePage.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import React, { FunctionComponent } from 'react';
+import './TutorialSinglePage.scss';
```
Comment (Jakeeyturner) [inline packages/blockchain-ui/src/components/pages/TutorialSinglePage/TutorialSinglePage.tsx:2]: This can be removed as it doesn't exist!

Comment (james-wallis) [reply packages/blockchain-ui/src/components/pages/TutorialSinglePage/TutorialSinglePage.tsx:2]: oops I removed the file but forgot the import...

**Review by Jakeeyturner — CHANGES_REQUESTED** [review]
- There are quite a few broken links, so it's worth double checking all of them in every tutorial
- Would be nice it we could apply a `cursor: pointer` style to each of the dropdown sections

File: packages/blockchain-extension/resources/tutorials/new-tutorials/basic-tutorials/a3.md [hunk 2]
```diff
@@ -30,6 +19,8 @@ Click on the IBM Blockchain Platform icon in the activity bar to show the blockc
 
 <img src="./images/a2.2-a3.2.png" alt="IBM Blockchain Platform side bar"></img>
 
+![IBM Blockchain Platform side bar](./images/a2.2-a3.2.png)
```
Comment (james-wallis) [inline packages/blockchain-extension/resources/tutorials/new-tutorials/basic-tutorials/a3.md:22]: need to remove this

File: packages/blockchain-extension/resources/tutorials/new-tutorials/basic-tutorials/a3.md [hunk 3]
```diff
@@ -43,7 +34,8 @@ We'll see later how this view also allows you to configure more realistic networ
 
 The required Hyperledger Fabric components are automatically downloaded and started when you select it.
 
-<img src="./images/bullet.png" alt="[]"></img> &nbsp;&nbsp;&nbsp;&nbsp; `A3.3`: &nbsp;&nbsp;&nbsp;&nbsp;
+<!-- <img src="./images/bullet.png" alt="[]"></img> -->
+![[]](./images/bullet.png) &nbsp;&nbsp;&nbsp;&nbsp; `A3.3`: &nbsp;&nbsp;&nbsp;&nbsp;
```
Comment (james-wallis) [inline packages/blockchain-extension/resources/tutorials/new-tutorials/basic-tutorials/a3.md:38]: need to remove this

File: packages/blockchain-ui/cypress/integration/tutorial_page.spec.ts [hunk 4]
```diff
@@ -2,9 +2,9 @@
 import ITutorialObject from '../../src/interfaces/ITutorialObject';
 chai.should();
 
-describe('Tutorial page', () => {
+describe.only('Tutorial page', () => {
```
Comment (james-wallis) [inline packages/blockchain-ui/cypress/integration/tutorial_page.spec.ts:5]: need to remove this

**Review by Jakeeyturner — APPROVED** [review]

**Pull request merged**